implants/lib/eldritch/src/assets/copy_impl.rs

```diff
@@ -61,17 +63,23 @@ pub fn copy(starlark_eval: &Evaluator<'_, '_>, src: String, dst: String) -> Resu
 
         if tmp_list.contains(&src_value.to_value()) {
             let env = Environment::from_extra(starlark_eval.extra)?;
-            let file_reciever = env.request_file(src)?;
+            let (tx, rx) = channel();
+            env.send(FetchAssetMessage { name: src, tx })?;
```

Comment:
Ohhh clever.
Eldritch instantiates a "callback channel" and passes it up noice 👌

Reply:
yeah this is how it was doing it before, just a bit more simplified